app/master/build_artifact.py

```diff
@@ -47,9 +47,10 @@ def write_timing_data(self, timing_file_path, timing_data):
             self._logger.debug('Created new timing file in {}', timing_file_path)
```

Comment:
I realized we can combine the above if block with the one below.

so

if not os.path.is.file(...) or len(self.get_failed_commands()) == 0:
...